.github/ISSUE_TEMPLATE/---report-an-issue.md

```diff
@@ -1,21 +1,24 @@
 ---
 name: "\U0001F41B Report an issue"
 about: Report an issue on parse-server
+title: ''
+labels: ''
+assignees: ''
 
 ---
 
-<!--- 
+We use GitHub Issues for reporting bugs with Parse Server.
 
-**We use GitHub Issues for reporting bugs with parse-server.**
+Make sure these boxes are checked before submitting your issue -- thanks for reporting issues back to Parse Server!
 
-If you have a *question*, you should join the [Parse Community's Discourse forum](https://community.parseplatform.org/c/parse-server).
+- [ ] This isn't a vulnerability disclosure, if it is please follow our [security policy](https://github.com/parse-community/parse-server/blob/master/SECURITY.md)
```

Comment:
My pedantic eye misses a period at the end of this item 🧐😁

Reply:
Done.